release-notes.md

```diff
@@ -3,6 +3,43 @@
 See [How to upgrade Anbox Cloud](https://discourse.ubuntu.com/t/upgrading-from-previous-versions/17750) or [How to upgrade the Anbox Cloud Appliance](https://discourse.ubuntu.com/t/upgrade-anbox-cloud-appliance/24186) for
 instructions on how to update your Anbox Cloud deployment.
 
+
+[Details=1.17.0]
+
+# 1.17.0 (February 15 2023)
+
+#### New features & improvements
+
+* Enabling work of Vulkan support for NVIDIA GPUs
+* Dynamic screen resolution support
+* Start and stop for containers in AMS
+* Various dashboard improvements
+* Experimental support for software accelerated AV1 encoding is now available behind a [feature flag](https://anbox-cloud.io/docs/ref/ams-configuration) (`experimental.force_av1_software_encoding`).
+* Containers now connect to Anbox Stream Agent instead of Anbox Stream Gateway for WebRTC signaling.
+* HTTPS fingerprint checks were added to the Anbox runtime for deployment scenarios where the Anbox Stream Agent is not deployed with a certificate signed by a trusted CA.
+* Initial support for NUMA has been added to AMS. If a system has multiple NUMA nodes, AMS will select a node based on the locality of an additionally allocated GPU. If a container is not allocated with a GPU or if `cpu.limit_mode` is set to `pinning`, no specific NUMA is being selected.
+* To allow scheduling containers for applications onto specific nodes in a LXD cluster, an application in AMS can now add a `node-selector` part into it's manifest. This will tell the scheduler in AMS to schedule a container for the application only onto a node matching the criteria of the node selector. The node selector currently supports tag based selection only.
```

Comment:
```suggestion
* To allow scheduling containers for applications onto specific nodes in a LXD cluster, an application in AMS can now add a `node-selector` part into its manifest. This will tell the scheduler in AMS to schedule a container for the application onto a node only if it matches the criteria of the node selector. The node selector currently supports tag based selection only.
```